Library/Homebrew/formula_installer.rb

```diff
@@ -1142,14 +1142,21 @@ def pour
       )
     end
 
-    tab.tap = formula.tap
+    # fill in missing/outdated parts of the tab
+    # keep in sync with Tab#to_bottle_json
+    tab.used_options = []
+    tab.unused_options = []
+    tab.built_as_bottle = true
     tab.poured_from_bottle = true
-    tab.time = Time.now.to_i
-    tab.head = HOMEBREW_REPOSITORY.git_head
-    tab.source["path"] = formula.specified_path.to_s
     tab.installed_as_dependency = installed_as_dependency?
     tab.installed_on_request = installed_on_request?
+    tab.time = Time.now.to_i
+    tab.compiler = DevelopmentTools.default_compiler
```

Comment:
On Linux I'd like to include `OS::Linux::Glibc.system_version` in the tab.

Reply:
@sjackman Can do, will just need to extend https://github.com/Homebrew/brew/blob/ab99855c12c02eebd185ca1402c33cccf62374f6/Library/Homebrew/development_tools.rb#L101-L106 on Linux.

Reply:
I did initially have something when I introduced this system but it was cut before merge.

I recall the desires were that it wasn't just a simple system version check but that it actually detected which glibc was used in the build (which may be the brewed glibc), which required more work that I didn't find the time for back then.

Reply:
My primary concern is for the bottles built on CI, which always use the host `glibc` and never the brewed `glibc`.

Reply:
Looking at it again, perhaps it's just as simple as:

```rb
Formula["glibc"].any_installed_version || OS::Linux::Glibc.system_version
```

if brewed glibc is guaranteed to be used when installed (`any_installed_version` will return nil if it's not installed).

And maybe another field to denote whether it was host or system glibc, so you can tell the difference.

Reply:
Looks good to me!

> And maybe another field to denote whether it was host or system glibc, so you can tell the difference.

Not necessary for the tab, I don't think, but knowing both is useful for `brew config`, and we have it there already.

Reply:
Is the revision number of brewed glibc important? If not I'd revise my suggestion to `Formula["glibc"].any_installed_version&.version` to strip the revision part.

Reply:
I don't expect the `revision` to be commonly useful.